src/frontend/next/src/theme/index.ts

```diff
@@ -5,9 +5,7 @@ export const lightTheme: Theme = createMuiTheme({
   palette: {
     type: 'light',
     primary: {
-      main: '#333E64',
-      contrastText: '#E5E5E5',
-      light: '#FFFFFF',
+      main: '#121D59',
     },
     secondary: {
       main: '#0589D6',
```

Comment:
Are we using this color on the new UI ?

Reply:
@PedroFonsecaDEV : It's supposed to be for the scrollToTop button. Let me update with the new color.